plugin.xml

```diff
@@ -101,11 +101,13 @@
         <config-file target="AndroidManifest.xml" parent="/manifest">
             <uses-permission android:name="android.permission.GET_ACCOUNTS" />
         </config-file>
-
+        <config-file target="AndroidManifest.xml" parent="/*">
+            <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
@fhls Please remove the permissions. The rest looks good to me.

Reply:
yes, permissions removed.